pkg/operator/operator.go

```diff
@@ -129,28 +129,35 @@ func (o *Operator) Run(stopc <-chan struct{}) error {
 		return nil
 	}
 
+	glog.V(4).Info("Waiting for initial cache sync.")
+	ok := cache.WaitForCacheSync(stopc, o.cmapInf.HasSynced)
```

Comment:
can this ever succeed before we start it?

Reply:
you're right, we need to run the informer first.